streams/src/test/java/org/apache/kafka/streams/state/internals/metrics/RocksDBMetricsRecorderTest.java

```diff
@@ -36,22 +38,23 @@
 import org.rocksdb.StatsLevel;
 import org.rocksdb.TickerType;
 
-import static org.easymock.EasyMock.anyObject;
-import static org.easymock.EasyMock.eq;
-import static org.easymock.EasyMock.expect;
-import static org.easymock.EasyMock.mock;
-import static org.easymock.EasyMock.niceMock;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.Assert.assertThrows;
-import static org.powermock.api.easymock.PowerMock.reset;
-import static org.powermock.api.easymock.PowerMock.createMock;
-import static org.powermock.api.easymock.PowerMock.mockStatic;
-import static org.powermock.api.easymock.PowerMock.replay;
-import static org.powermock.api.easymock.PowerMock.verify;
-
-@RunWith(PowerMockRunner.class)
-@PrepareForTest({RocksDBMetrics.class, Sensor.class})
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
> @RunWith(MockitoJUnitRunner.StrictStubs.class)

I explicitly added this, though, shouldn't it be default behaviour already since 3.x version of Mockito (we are on 4.4)?

> this test would fail even without explicit verify with following message:

That is fair. We can either specify the expectation using a stub and no verify OR we can avoid the stud for void method and verify the call. I opted for latter in latest revision.

Reply:
> I explicitly added this, though, shouldn't it be default behaviour already since 3.x version of Mockito (we are on 4.4)?

You're right! It is the default behavior and it works as expected and described in the docs. It's not needed to use `@RunWith(MockitoJUnitRunner.StrictStubs.class)`. I apologize.

Before adding the initial comment about `StrictStubs` usage I double checked the docs, and I have updated few test cases with stubs that are never invoked, and tests were still passing 😕 Only once I added explicit `StrictStubs` tests started to fail (that's the reasoning behind initial comment, but it seems something wasn't working as expected on my machine 😅 )

Reply:
You were not completely wrong. I have found the following idiosyncrasies:
1. `StrictStubs` does not verify invocation of mocks for `mockstatic` cases. You need an explicit `verify` call to do that.
2. When you run a test `StrictStubs` does not perform verification but when you run all together, it does ensure that all stubs are being invoked.

I think it's safer to keep `StrictStubs` for now, so let's keep the change as you suggested.